pyart/graph/radardisplay.py

```diff
@@ -708,7 +774,19 @@ def plot_colorbar(self, mappable=None, field=None, label=None, cax=None,
             if field is None:
                 field = self.plot_vars[-1]
             label = self._get_colorbar_label(field)
-        cb = fig.colorbar(mappable, ax=ax, cax=cax)
+
+        if orient is None:
+            orient = 'vertical'
+
+        # Find the axes locations to set colorbar
+        if cax is None:
```

Comment:
No idea, I just copy what was already in RadarDisplay_airborne, as it was an enhancement of `plot_colorbar` i didn't really question anything